src/PagingControl/PagingControl.js

```diff
@@ -256,6 +255,7 @@ module.exports = kind(
 			return;
```

Comment:
Is there any reason we're receiving the sender, event here?  Should we clean this up and revert the spotlightBlurred below?